spec/requests/regions_spec.rb

```diff
@@ -42,10 +42,10 @@
   end
 
   describe "/api/regions/:id/settings" do
-    let(:region_number) { ApplicationRecord.my_region_number + 1 }
```

Comment:
Needed to make this change too because zones for the current region are created in the EVM test setup. From what I can tell, these test do not rely on having a new region. They just need a region.